policyengine_us/parameters/gov/states/nm/tax/income/exemptions/social_security_income/income_limit.yaml

```diff
@@ -0,0 +1,28 @@
+description: New Mexico provides the secial security income exemption for filers with adjusted gross income at or below this amount.
+metadata:
+  label: New Mexico social security income exemption income limit
+  period: year
+  unit: currency-USD
+  breakdown:
+  - filing_status
+  reference:
+  - title: 2022 New Mexico personal income tax form packet ADJ-7 Exemption for social security income.
+    href: https://klvg4oyd4j.execute-api.us-west-2.amazonaws.com/prod/PublicFiles/34821a9573ca43e7b06dfad20f5183fd/1afc56af-ea90-4d48-82e5-1f9aeb43255a/PITbook2022.pdf#page=52
+  - title: New Mexico Statutes, 7-2-5.14. Exemption; social security income.
```

Comment:
That it came into effect in 2022 seems clear from the law, but could you cite a 2021 tax form too to show its absence?